infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java

```diff
@@ -275,30 +273,21 @@ private KubernetesNamespaceMeta asNamespaceMeta(Namespace namespace) {
    * the context.
    *
    * @param identity the identity of the workspace runtime
+   * @param userName the user name
    * @return true if the namespace can be created, false if the namespace is expected to already
    *     exist
    * @throws InfrastructureException on failure
    */
-  protected boolean canCreateNamespace(RuntimeIdentity identity) throws InfrastructureException {
+  protected boolean canCreateNamespace(RuntimeIdentity identity, String userName)
```

Comment:
I also thought about that, but this would require quite a few changes in the obsolete classes like PluginBrokerManager which are planned to be removed completely in Phase 2 of the db removal procedure. The goal of this PR is to make all the stored data in Postgres irrelevant. In the future PR, major cleanup  / refactoring is going to happen when the whole DAO layer will be removed completely (currently targeted for 7.61)